generation/hermetic_library_generation.sh

```diff
@@ -89,28 +87,14 @@ fi
 git show "${target_branch}":"${generation_config}" > "${baseline_generation_config}"
 config_diff=$(diff "${generation_config}" "${baseline_generation_config}")
 
-# bind docker volume to include the repository in docker running environment.
-if [[ $(docker volume inspect ${volume_name}) != '[]' ]]; then
-  docker volume rm ${volume_name}
-fi
-docker volume create \
-  --name ${volume_name} \
-  --opt "type=none" \
-  --opt "device=$(pwd)" \
-  --opt "o=bind"
 # run hermetic code generation docker image.
 docker run \
   --rm \
-  -v "${repo_volumes}" \
-  -v /tmp:/tmp \
-  -v /var/run/docker.sock:/var/run/docker.sock \
-  -e "RUNNING_IN_DOCKER=true" \
-  -e "REPO_BINDING_VOLUMES=-v ${repo_volumes}" \
+  -u "$(id -u):$(id -g)" \
```

Comment:
I guess this is the command enables us to modify/delete the docker generated files?

Reply:
yes, including ownership takeover performed by the container